src/java/us/kbase/mobu/initializer/ModuleInitializer.java

```diff
@@ -125,7 +124,7 @@ public void initialize(boolean example) throws Exception {
 		templateFiles.put("module_readme_ui", Paths.get(moduleDir, "ui", "README.md"));
 		templateFiles.put("module_readme_test", Paths.get(moduleDir, "test", "README.md"));
 		templateFiles.put("module_readme_docs", Paths.get(moduleDir, "docs", "README.md"));
-		templateFiles.put("module_readme_data", Paths.get(moduleDir, "data", "README.md"));
+		//templateFiles.put("module_readme_data", Paths.get(moduleDir, "data", "README.md"));
```

Comment:
I was thinking about removing things that aren't used, but realized this is used for ref data.  I'll leave this and the other ui/widgets placeholder directories for now.